templates/docs/patterns/lists.md

```diff
@@ -53,12 +53,20 @@ View example of the ticked divided list pattern
 
 ### Responsive divider
 
+<span class="p-label--updated">Updated</span>
+
 A responsive divider inserts divider lines between sections of content. On small screens (up to `$breakpoint-medium`), the divider lines appear horizontally. On screens bigger than `$breakpoint-medium`, the divider lines appear vertically, centered in the column gutters.
 
 <div class="embedded-example"><a href="/docs/examples/patterns/lists/divider/" class="js-example">
 View example of lists with a responsive divider
 </a></div>
 
+The utility class `.is-dark` can also be applied:
```

Comment:
I copy-paste the theming section form one of the other examples where we describe it, and reword it to apply to this component. E.g. here: https://vanilla-framework-3728.demos.haus/docs/patterns/navigation#theming